src/App/Pages/Accounts/TwoFactorPage.xaml.cs

```diff
@@ -195,5 +183,17 @@ private async Task TwoFactorAuthSuccessAsync()
                 Application.Current.MainPage = new TabsPage(_appOptions, previousPage);
             }
         }
+
+        private void Token_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            if (string.IsNullOrWhiteSpace(e.NewTextValue))
+            {
+                ContinueButton.IsEnabled = false;
```

Comment:
Please use a property like `EnableContinue`, which is bound to the `IsEnabled` property of the button in the XAML. This is more consistent with how the other buttons here are handled, and how you've handled `IsVisible` for this button.